modules/solid_mechanics/doc/content/modules/solid_mechanics/1d_elastic_waves.md

```diff
@@ -1,5 +1,93 @@
 # Frequency Domain Analysis
 
+For frequency domain analysis for Solid Mechanics MOOSE has two main methods
+modal analysis and frequency response functions. The following example will go
+through both options.
+
+
+## Modal Analysis
+
+Modal analysis is a technique used in structural dynamics to determine the
+natural frequencies (eigenvalues) and mode shapes (eigenvectors) of a structure.
+The natural frequencies represent the frequencies at which the structure tends
+to vibrate when subjected to a disturbance, while the mode shapes describe the
+corresponding deformed shape of the structure at each natural frequency. Modal
+analysis provides valuable insights into the dynamic behavior of a structure and
+is often used for design, optimization, and troubleshooting purposes.
+
+The general eigenvalue problem that needs to be solved for modal analysis is given by:
+\begin{equation}
+(\mathbf{A} - \omega^2 \mathbf{B})\mathbf{u} = 0
+\end{equation}
+where $\mathbf{A}$ holds the "noneigen" kernels, $\mathbf{B}$ holds the "eigen" kernels, $\omega$ are the natural frequencies, and $\mathbf{u}$ are the mode shapes.
+In MOOSE, modal analysis is set up using the Eigenvalue Executioner. The
+Eigenvalue System allows you to define which kernels go into which matrix. It's important
+to note that in modal analysis, Neumann boundary conditions are
+ignored since we are interested in the free vibration response of the structure.
+Only Dirichlet boundary conditions are considered.
+
```

Comment:
Will you move lines 136-153 here that describe the analytic solution.  Then your results will follow it.  The analytic results are $\omega_{1a}=$509Hz\\
$\omega_{1b}=$763Hz. Do you know if you are using the same coarse mesh as the coarse mesh FRF results below.  If so, natural frequencies are much better, you should mention that.